.github/cluster-upstream.yml

```diff
@@ -0,0 +1,21 @@
+# Docker-compose setup used during tests
```

Comment:
How does this differ from the other docker-compose setup file still here?

Reply:
Only difference is that we have the Dask conda nightlies specified in `EXTRA_CONDA_PACKAGES` here - I would've liked to handle this more gracefully with some conditionals within the docker-compose file, but it seems like conditional variable setting isn't really an easy thing to do within compose files.

Open to other options here - I imagine just replacing the files with the equivalent `docker run ...` commands would probably be easier, though that comes at the cost of making local debugging somewhat more difficult.

Reply:
Docker compose does support env var substitution so perhaps the extra packages could be configured that way?

https://docs.docker.com/compose/environment-variables/